crates/ef-testing/src/evm_sequencer/evm_state.rs

```diff
@@ -191,6 +197,50 @@ impl EvmState for KakarotSequencer {
             Into::<FieldElement>::into(nonce).to_bytes_be(),
         ))
     }
+
+    fn get_code_at(&mut self, evm_address: &Address) -> StateResult<Bytes> {
+        let starknet_address = compute_starknet_address(evm_address);
+
+        let bytecode_len = (&mut self.0.state).get_storage_at(
```

Comment:
will this be 0 for an EOA?

Reply:
Yes, the EOA will not have the storage var set for bytecode_len_, hence this will always return 0 for an eoa